src/status_im/contexts/chat/home/chat_list_item/view.cljs

```diff
@@ -184,12 +184,21 @@
      {:customization-color color
       :size                :size-32}]))
 
+(defn- notification-layout
+  [child]
+  (when (seq child)
+    [rn/view
+     {:style {:flex-grow       1
+              :justify-content :center
+              :margin-left     8}}
+     (into [rn/view {:style style/notification-container}] [child])]))
```

Comment:
Hey @seanstrom , I think the `child` var seems a bit strange here.

why are we asking `(seq child)`? are we trying to check if it's a non-empty seq? should we name it `children` instead? or are we checking if it's hiccup? if so, probably `vector?` works better :thinking: 

Keep in mind that when not `(seq child)` this component will render nothing, so please consider always rendering the content but only call the component when `(seq child)`.


Additionally, the `into` at the end seems unnecessary, it seems it behaves the same as:
```clojure
[rn/view {:style style/notification-container}
 child]
```

Reply:
I'll refactor the code to avoid these weird usages of `(seq child)`, but the original intent was to keep some other code dry by using `notification-layout` around a `cond` expression. 

Initially I thought we need to use `into`, but I think you're right, I'll remove it 👍